<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>538</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>lab2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="Frame" name="drawArea1">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Frame" name="drawArea2">
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::NoDrag</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Frame" name="drawArea3">
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::NoDrag</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Frame" name="drawArea4">
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::NoDrag</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Zoom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="zoomSB">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>3.500000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="drawLines">
        <property name="text">
         <string>Draw lines</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="xyzZone">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Coords</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="layout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="xSB">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="minimum">
                <double>-9.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="ySB">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="minimum">
                <double>-9.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="zSB">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="minimum">
                <double>-9.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Frame</class>
   <extends>QGraphicsView</extends>
   <header>Frame.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
